enable/api.py

```diff
@@ -283,6 +283,7 @@
 try:
     import kiwisolver
 except ImportError:
+    print("kiwisolver not imported successfully imported. Please check it is installed.")
```

Comment:
@homosapien-lcy Please revert this change. As discussed in the comments, for non-kiwisolver users, we don't want a message printed every time `enable.api` is imported.